components/Activitylist.js

```diff
@@ -1,21 +1,55 @@
 import { activityList } from "@/dummyList/activityList";
-import { styled } from "styled-components";
+import styled from "styled-components";
 import { StyledHeadline } from "./Dashboard";
+import Form from "./Form";
+import { uid } from "uid";
+import useLocalStorageState from "use-local-storage-state";
 
 export default function Activitylist() {
+  // the defaultValue is connected to the dummyList. This is helpful in the future for the mongoDB connection
+  const [activityCards, setActivityCards] = useLocalStorageState(
+    "activityList",
+    {
+      defaultValue: activityList,
+    }
+  );
+
+  function handleSubmitActivity(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    const inputData = {
```

Comment:
maybe you can find a more fitting Name than inputData, but for now its fine.

Reply:
for Sure! I'll changed that to "newActiviry"